DISCLAIMER-WIP

```diff
@@ -0,0 +1,19 @@
+Apache XTable (incubating) is an effort undergoing incubation at The Apache
```

Comment:
this seems consistent with what we had for Hudi back then
https://github.com/apache/hudi/blob/release-0.5.2-incubating/DISCLAIMER